packages/repo-doctor/src/toolbelt.js

```diff
@@ -0,0 +1,16 @@
+const fromEntries = entries => {
+  const res = {}
+  for (let entry of entries) {
+    res[entry[0]] = entry[1]
+  }
+  return res
+}
+
+const keyBy = (values, fn) => {
+  return fromEntries(values.map(v => [fn(v), v]))
+}
+
+module.exports = {
+  fromEntries,
+  keyBy
```

Comment:
memoize could probably go in here as well, through i'm surprised we can't use lodash for any of these.

Reply:
Yep, I don't know why I recoded these functions instead of using lodash.